packages/lodestar/src/node/nodejs.ts

```diff
@@ -148,7 +148,7 @@ export class BeaconNode {
     // eth1 is started in chain
     await this.chain.start();
     await this.network.start();
-    this.sync.start();
+    await this.sync.start();
```

Comment:
Not awaiting the sync was a typo or there was a reason?

Reply:
sync start method returns when we are in sync. We need some sync initialization for api and chores I think

Reply:
Related to the discussion above, seems that `sync.start()` won't resolve until Eth1 blocks finish processing. Is this correct?

Reply:
hmm that shouldn't be connected but this await will prevent api and chores from starting which should be fatal

Reply:
Then what should we do in this case? IMO the root issue here is that `sync.start()` should resolve once the essential initialization is done

Reply:
We should refactor the sync module to respect the "start should resolve quickly" interface.
In this PR, just eslint-disable and document.

Reply:
Opened follow up issue https://github.com/ChainSafe/lodestar/issues/1196